docs/content/manual/_incoming/2376-2379-delete-vm-template-default-version.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Delete VM template default version
+---
+
+* Related issues: [#2376](https://github.com/harvester/harvester/issues/2376) [BUG] Cannot delete Template
+* Related issues: [#2379](https://github.com/harvester/harvester/issues/2379) [backport v1.0.3] Cannot delete Template
+
+## Category: 
+* VM Template
+
+## Verification Steps
+1. Go to advanced -> tempaltes
```

Comment:
should be `template`

Reply:
Thanks for the check, update this with Advanced -> Templates